Documentation/ceph-common-issues.md

```diff
@@ -11,73 +11,127 @@ Many of these problem cases are hard to summarize down to a short phrase that ad
 If after trying the suggestions found on this page and the problem is not resolved, the Rook team is very happy to help you troubleshoot the issues in their Slack channel. Once you have [registered for the Rook Slack](https://slack.rook.io), proceed to the General channel to ask for assistance.
 
 ## Table of Contents
-- [Troubleshooting Techniques](#troubleshooting-techniques)
-- [Pod using Rook storage is not running](#pod-using-ceph-storage-is-not-running)
-- [Cluster failing to service requests](#cluster-failing-to-service-requests)
-- [Monitors are the only pods running](#monitors-are-the-only-pods-running)
-- [PVCs stay in pending state](#pvcs-stay-in-pending-state)
-- [OSD pods are failing to start](#osd-pods-are-failing-to-start)
-- [OSDs are not created on my devices](#osd-pods-are-not-created-on-my-devices)
-- [Node hangs after reboot](#node-hangs-after-reboot)
-- [Rook Agent modprobe exec format error](#rook-agent-modprobe-exec-format-error)
-- [Rook Agent rbd module missing error](#rook-agent-rbd-module-missing-error)
-- [Using multiple shared filesystem (CephFS) is attempted on a kernel version older than 4.7](#using-multiple-shared-filesystem-cephfs-is-attempted-on-a-kernel-version-older-than-47)
-- [Activate log to file for a particular Ceph daemon](#activate-log-to-file-for-a-particular-ceph-daemon)
-- [Flex storage class versus Ceph CSI storage class](#flex-storage-class-versus-ceph-csi-storage-class)
+
+* [Ceph Common Issues](#ceph-common-issues)
+  * [Table of Contents](#table-of-contents)
+* [Troubleshooting Techniques](#troubleshooting-techniques)
+  * [Ceph Tools](#ceph-tools)
```

Comment:
This is a very detailed table of contents. Do you think it's helpful? My impression is that it very verbose. But if it's generated, it could be worth it.

Reply:
Yep, the table of contents is generated. Right now though a plugin for VSSCode is doing that. I'll strip it down to what is needed and take a look at the rook.github.io repo if we can have a toc toggleable with the front matter (the parameters between the `---` at the top of the doc files).